k8s/src/main/scala/io/buoyant/k8s/ClientConfig.scala

```diff
@@ -0,0 +1,59 @@
+package io.buoyant.k8s
+
+import com.twitter.finagle.{Filter, Http, Stack}
+import com.twitter.finagle.http.{Request, Response}
+import io.buoyant.k8s
+import scala.io.Source
+
+/**
+ * Meant to be implemented by configuration classes that need to
+ * produce a k8s API client.
+ */
+trait ClientConfig {
+  import ClientConfig._
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/ver/linkerd/file/2415518678/issues/source?bid=3236810&fileBranchId=3236828#l13)